example/rater/generated_answer.ipynb

```diff
@@ -0,0 +1,355 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Use AutoRater to compare a generated answer to grounding answer from a Jupyter Notebook\n",
```

Comment:
Can you change the title to "Use `AutoRater` to Compare Answers to a Given Question"